### ISSUE TITLE
`std_errors` for `absorbed_effects`

### ISSUE THREAD

**Issue by Alalalalaki** [bashtage/linearmodels#363]
Is it possible to get `std_errors` for `absorbed_effects`?

**Comment by bashtage** [comment]
No.  If you need standard errors, then you would need to include any effects as regressors. If these effects are high dimensional, then it is likely that inference would not be standard since there would usually only be finitely many observations for each effect.

**Comment by bashtage** [comment]
Closing since I think I answered as best as can be done.  Happy to continue the discussion though if helpful.

**Comment by Alalalalaki** [comment]
@bashtage I am not looking for the inference. The thing is I need to calculate the variance of the absorbed effects (which is used as a term of the variance decomposition of the original regression), which is biased, and one simple way of adjustment is to use the HC standard errors of these absorbed effects. This adjustment is not perfect because the finite obs for each effect as your said, but I want to use it as an initial try and a baseline.

I haven't tried including these effects as regressors, because these effects are high dimensional and I heard there is a computational problem of inverting the matrix.

**Comment by bashtage** [comment]
There are a number of challenges here:

1. The absorbed effects only absorb the span of the effects, rather than the actual effects. Specifically, the absorbed effects are not required to have full rank.  For example, if you include 3 dummies, the effects absorbed will use all dummies from all 3 effects, which we know is a singular array.  This is fine because LSMR is regularized enough to provide the correct solution in terms of the span of the regressors even when singular.
2. Standard errors calculation requires (X'X)^{-1}, which is potentially huge and incredibly slow to compute. 
3. Interpretation is very difficult without assumptions like homoskedasticity and exact normality (so that a standard error is meaningful since in most cases it is only really meaningful when combined with asymptotic normality).

**Comment by bashtage** [comment]
If you knew your effects where orthogonal, then you should be able to do something.  This would seem quite special purpose though.
